libs/design-system/indicators/progress-spinner/src/lib/progress-spinner.component.scss

```diff
@@ -0,0 +1,29 @@
+@use '@angular/material' as mat;
```

Comment:
Remove this import

Use --mdc-circular-progress-active-indicator-color to set color instead

Reply:
This is the new angular 19 way